### PR TITLE
Adding Cohort 3 Week 2 slides and link to video in readme

### PR DESCRIPTION
**Pull request by aleighbrown**
Wasn't sure how to link to the slides properly until uploaded

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,10 +56,10 @@ This repo contains 3 main folders:
 
 <hr>
 
-*Cohort 3 [Europe/Africa]: (started 2020-08-04) Tuesdays, 19:15 CET (CEST)*
+*Cohort 3 [Europe/Africa]: (started 2020-08-04) Tuesdays, 19:30 CET (CEST)*
 - 2020-08-11 Chapter 2 - Names and values [Anna-Leigh](https://twitter.com/annaleighbrown2)
-- 2020-08-18 Chapter 3 - Vectors 
-- 2020-08-25 Chapter 4 - Subsetting [Shel] and [Alan Kinene] 
+- 2020-08-18 Chapter 3 - Vectors
```
Comment (tanho63) [inline README.md:61]: Can you delete this entry from "upcoming meetings"? Have been reserving these for meetings that have yet to be completed 😎

File: README.md [hunk 2]
```diff
@@ -84,6 +84,7 @@ Chapter 2 - Names and values
 
 - Cohort 1: [Asmae Toumi](https://twitter.com/asmae_toumi) ([slides](https://r4ds.github.io/bookclub-Advanced_R/Presentations/Week02/Cohort1/Chap2slides.html#1))
 - Cohort 2: [Josh Pohlkamp-Hartt](https://twitter.com/jpohlkamphartt) ([slides](https://r4ds.github.io/bookclub-Advanced_R/Presentations/Week02/Cohort2_America/Chapter2Slides.html#1) | video)
+- Cohort 3: [Anna-Leigh Brown](https://twitter.com/annaleighbrown2)([video](https://youtu.be/-bEXdOoxO_E))
```
Comment (tanho63) [inline README.md:87]: Were the slides made in a ~powerpoint-like system? Might be helpful to upload those, but if not PDF is fine. You can link it with 

```
[slides](https://github.com/r4ds/bookclub-Advanced_R/blob/main/Presentations/Week02/Cohort3_Africa_Europe/Advanced%20R%20Chapter%202%20Names%20and%20values.pdf)

Comment (aleighbrown) [reply README.md:87]: Made in Google Docs, happy to link directly to Google Docs rather than PDF

File: README.md (hunk 1 above)
Comment (aleighbrown) [reply README.md:60]: Done!

**Pull request merged**